Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:16.13.2
+
+WORKDIR /app
+
+COPY package*.json ./
+RUN npm i
+
+COPY . .
+COPY ./dist ./dist
+
```

Comment:
how about running `migrate:latetst` or whatever command it is, to ensure DB schemas are up-to-date before service start